### PR TITLE
Release post for v4.3.0

### LINKED ISSUE #9095: Time for a new release
Hello, maintainers! :wave:

By my calculations, it's time for a new release of jekyll. There have been 297 commits since the last release, [v4.2.2](https://github.com/jekyll/jekyll/releases/tag/v4.2.2).

What else is left to be done before a new release can be made? Please make sure to update History.markdown too if it's not already updated.

Thanks! :revolving_hearts: :sparkles:

### PR DESCRIPTION
**Pull request by ashmaroli**
- This is a 🔦 documentation change.

## Summary

Its about time v4.3.0 got shipped.
Setting the date to be on Thursday, Oct 20.

## Context

Closes #9095

### CODE REVIEW

File: docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+### Announcement
+
+I would like to inform you that following this release, Jekyll will start developing towards a v5.0 milestone that will
+**definitely contain breaking changes**. I have set up a [tentative roadmap at the GitHub repository][roadmap] to give everyone
+a glimpse of the PROBABLE OUTCOME. Towards that end, we will no longer accept documentation fixes on `master`. The `4.3-stable`
+branch will be used to build and deploy the site for https://jekyllrb.com.
+
+Jekyll 3.x series is now under security-maintennce phase. Only security patches will be released when necessary.
```
Comment (yboulkaid) [inline docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown:77]:
```suggestion
Jekyll 3.x series is now under security-maintenance phase. Only security patches will be released when necessary.
```

File: docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Jekyll 4.x series will continue recieving bug-fixes and security-patches only. Depending on the state of progress towards v5.0,
+there will be *at least* one minor version release serving as a transistionary version containing deprecations and bridge code
```
Comment (yboulkaid) [inline docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown:80]:
```suggestion
there will be *at least* one minor version release serving as a transitionary version containing deprecations and bridge code
```

File: docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+to ease eventual major upgrade.
```
Comment (yboulkaid) [inline docs/_posts/2022-10-20-jekyll-4-3-0-released.markdown:81]:
```suggestion
to ease eventual major upgrades.
```

**Comment by ashmaroli** [comment]
@jekyllbot: merge +doc

**Pull request merged**